infra/ibmcloud/terraform/k8s-s390x-build-cluster/bastion.tf

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
the use of `k8s-*` throughout the file names, resource names, etc. seems to be redundant (i.e. - its defined in a K8s repo/org so its assumed to be for K8s, no need to re-state that in the names)